docs/source/ref/models/fields.rst

```diff
@@ -420,3 +420,90 @@ These indexes use 0-based indexing.
 .. admonition:: Forms are not supported
 
     ``PolymorphicEmbeddedModelArrayField``\s don't appear in model forms.
+
+.. _encrypted-fields:
+
+Encrypted fields
+================
+
+.. versionadded:: 5.2.0b2
+
+Encrypted fields are subclasses of Django's built-in fields and can be used to
+store sensitive data with MongoDB's :ref:`Queryable Encryption
+<queryable-encryption>` feature. They are subclasses of Django's
+built-in fields before storing it in the database.
+
++----------------------------------------+------------------------------------------------------+
+| Encrypted Field                        | Django Field                                         |
++========================================+======================================================+
+| ``EncryptedBigIntegerField``           | :class:`~django.db.models.BigIntegerField`           |
```

Comment:
I suggested not to include EncryptedBigIntegerField since it's identical to EncryptedIntegerField. I guess it's fine in case someone wants to blindly add "Encrypted" to their existing fields and doesn't know about this.

EncryptedDurationField can be added.

Reply:
removed